src/Silverback.Integration.Kafka/Messaging/Messages/KafkaStatisticsEvent.cs

```diff
@@ -1,23 +1,36 @@
 ﻿// Copyright (c) 2020 Sergio Aquilini
 // This code is licensed under MIT license (see LICENSE file for details)
 
+using Newtonsoft.Json;
+using Silverback.Messaging.Messages.Statistics;
+
 namespace Silverback.Messaging.Messages
 {
     /// <summary>
     ///     The event fired when statistics are received. Statistics are provided as a JSON formatted string
     ///     as defined here: https://github.com/edenhill/librdkafka/blob/master/STATISTICS.md
+    ///     and also as <see cref="KafkaStatistics"/> object.
     /// </summary>
     /// <remarks>
     ///     You can enable statistics and set the statistics interval using the <c>StatisticsIntervalMs</c>
     ///     configuration parameter (disabled by default).
     /// </remarks>
     public class KafkaStatisticsEvent : IKafkaEvent
     {
-        public KafkaStatisticsEvent(string statistics) => Raw = statistics;
+        public KafkaStatisticsEvent(string statistics)
+        {
+            Raw = statistics;
+            Statistics = JsonConvert.DeserializeObject<KafkaStatistics>(statistics);
```

Comment:
Would it be possible to deserialize in the property getter (only once), to avoid paying the cost if nobody is even listening to the event? (it's not a big deal since we aren't blocking anything, but we can still save some CPU cycles)

Reply:
Also, if the deserialisation ever fails, it will throw an exception in the user code, instead of just skipping the publish of the event.